### PR TITLE
SW-2397 / SW-2405 More empty state updates for plants dashboard 

### PR DESCRIPTION
**Pull request by karthikbtf**
- set aspect ratio of map and charts
- set height of charts
- don't show plot/species chart if site has no plots
- fix some margin bottom issues

Terraware App 2022-11-30 15-33-00

Terraware App 2022-11-30 15-28-36

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -81,6 +81,10 @@ const useStyles = makeStyles((theme: Theme) => ({
       paddingBottom: theme.spacing(25),
       paddingTop: '96px',
     },
+    '& > .moreScreen': {
```
Comment (karthikbtf) [inline src/App.tsx:84]: @joshbuchea I added this as a way to override the 200px margin bottom. Hoping this looks ok.

File: src/components/PlantingSites/PlantingSiteCreate.tsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ export default function CreatePlantingSite(props: CreatePlantingSiteProps): JSX.
   };
 
   return (
-    <TfMain>
+    <TfMain moreScreen={true}>
```
Comment (constanzauanini) [inline src/components/PlantingSites/PlantingSiteCreate.tsx:132]: I think I missed in visual changes what this moreScreen prop does?

Comment (karthikbtf) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:132]: Every page with TfMain has 200px bottom padding which is a lot. This option reduces bottom padding to 80px so we can show a taller map

Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:132]: Thanks for the explanation!

**Pull request merged**